pkg/html/urlutil/urls.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package urlutil
+
+import "net/url"
+
+func AmendURL(baseURL, key, value string) string {
```

Comment:
`SetParam` may be the better candidate.
There are 2 reasons:
1. Package name contains `url`.
2. It better explains what part of the `url` will be changed.